src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from 'react'
+import { Grid, Avatar, Typography, Button, Chip, Card, CardContent, Stack } from '@mui/material'
+import BusinessIcon from '@mui/icons-material/Business'
+import LocationOnIcon from '@mui/icons-material/LocationOn'
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+
+
+interface JobCardProps {
+    title: string;
+    company: string;
+    logoUrl: string;
+    jobPostingUrl: string;
+    location:string,
+    salary: string;
+    publicationDate:string;
+    tags: string[];
+}
+
+const JobCard: FC<JobCardProps> = ({ title, company, logoUrl, jobPostingUrl, location, salary, publicationDate, tags }) => {
+    const handleReadMoreClick = () => {
+        if (jobPostingUrl) {
+            window.open(jobPostingUrl, '_blank', 'noopener,noreferrer')
+        } else {
+            console.error('No URL provided')
+        }
+    }
+
+    return (
+        <Stack spacing={1} alignItems="center">
```

Comment:
Let's remove the `Stack` from here. Stacks are for 2-dimensional arrangements of multiple items. Since we only have one card inside, we can make that the outermost component.